lib/compute/jenkins-main-node.ts

```diff
@@ -299,7 +300,7 @@ export class JenkinsMainNode {
                   auto_removal: true,
                   log_stream_name: 'jenkins.log',
                   //  2021-07-20 16:15:55.319+0000 [id=868]   INFO    jenkins.InitReactorRunner$1#onAttained: Completed initialization
-                  timestamp_format: '%Y-%m-%d %H:%M:%S.%f%z',
+                  timestamp_format: '%Y-%m-%d %H:%M:%S.%%z',
```

Comment:
I havent see a time format using double %%, nor shown in their manual page.
Is this a miss after all?
Thanks.

Reply:
Fixing this, there should be no change on this line.